src/com/google/net/OAuth2.js

```diff
@@ -117,8 +117,8 @@ foam.CLASS({
 
```

Comment:
Ignore file: Feature required universal HTTPRequest. Will merge in #248 after it's reviewed.